pkgs/applications/graphics/meshlab/default.nix

```diff
@@ -1,28 +1,36 @@
-{ fetchFromGitHub, libGLU, llvmPackages, qtbase, qtscript, qtxmlpatterns }:
+{ mkDerivation, lib, fetchFromGitHub
+, libGLU
+, qtbase
+, qtscript
+, qtxmlpatterns
+, lib3ds
+, bzip2
+, muparser
+, levmar
+}:
 
 let
-  meshlabRev = "d596d7c086c51fbdfb56050f9c30b55dd0286d4c";
-  vcglibRev = "6c3c940e34327322507c703889f9f1cfa73ab183";
+  meshlabRev = "25f3d17b1d1d47ddc51179cb955f3027b7638745";
+  vcglibRev = "910da4c3e310f2e6557bd7a39c4f1529e61573e5";
```

Comment:
vgclib looks like a standalone library (used in few other applications) to have its own build derivation, is there a reason to keep it in the same build as meshlab?

Reply:
Yes, meshlab upstream does not seem to follow a proper release process, and the other dependencies of `vcg` may no longer work when we upgrade it for the purpose of meshlab.

It can often be a challenge to find the commit of `vgclib` that works with meshlab, as from observation developers seem to just push when they have a working one in their local tree; until that process is improved, I think we should use a meshlab-specific vcg here.

I'll add a comment.